### PR TITLE
ft: add stats to healthcheck route

### PR DESCRIPTION
**Pull request by rahulreddy**
This commit introduces stats for healthcheck route. On a successful healthcheck, the response
includes the total number of requests received and the number of 500s occured over a sampling
duration. The current default for sampling duration is 30 seconds.

### CODE REVIEW

**Comment by rahulreddy** [comment]
@ironman-machine

File: Healthchecks.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+ Server is up and running
+
+- 500 Internal Server error
+
+ Server is experiencing an Internal Error
+
+- 400 Bad Request
+
+ Request is not allowed due to IP restriction
+
```
Comment (dora-korpar) [inline Healthchecks.md]: The healthcheck can also return a 403 Bad Request error

**Comment by rahulreddy** [comment]
@ironman-machine again

File: Healthchecks.md (hunk 1 above)
Comment (rahulreddy) [inline Healthchecks.md]: I have added the change.

**Comment by rahulreddy** [comment]
@ironman-machine this time with the proper config file

**Comment by rahulreddy** [comment]
@ironman-machine config setup was not ideal

**Comment by rahulreddy** [comment]
@ironman-machine ...

**Comment by rahulreddy** [comment]
@ironman-machine ...

**Comment by rahulreddy** [comment]
This needs to be merged after https://github.com/scality/Integration/pull/316

**Comment by rahulreddy** [comment]
Squashed all commits

**Comment by rahulreddy** [comment]
@ironman-machine

**Comment by rahulreddy** [comment]
@ironman-machine one last time

**Pull request merged**